Shubham_Dalal/HTML/ASSIGNMENT6/Shubham_Ass6_ques1.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment6</title>
+        <link rel="stylesheet" href="assign6_style.css" />
+    </head>
+    <body>
+        <div class="heading1">
+            <h2>MI-11 Intern HTML Assessment</h2>
+        </div>
+        <hr />
+        <div class="heading2">
+            <h3><i>Personal Details</i></h3>
+        </div>
+        <form class="form1">
+            <label>First Name: &nbsp;</label>
+            <input type="text" name="name" placeholder="First Name" /><br /><br />
```

Comment:
placeholders second words 1st letter should be small not capital like First name, so make this change in all input boxes.